data/schedule.py

```diff
@@ -7,12 +7,15 @@
 from data import status
 from data.game import Game
 from data.update import UpdateStatus
+from datetime import timedelta
 
 GAMES_REFRESH_RATE = 5 * 60
 
 
 class Schedule:
     def __init__(self, date, config):
+        date -= timedelta(hours=4)
+
```

Comment:
Yes.  I referred to this issue in my initial comments above.  I was working on some of this code, and after midnight I could no longer see the live games that were still in progress on the west coast.  It started rotating through the future "current day" games.  

I found the code that determines which day's games to query.  Moving the cutoff fixes the problem.  I realize that 4 hours is arbitrary, but it's unlikely that any games will continue past 1 AM PST.    I realize that this looks hacky - I could make it a parameter.  

This would have a side effect that the "final scores" from the previous day would show until 4 AM EST, but I don't think that's as much of an issue as getting live games cut off at midnight (9 PM PST).

Reply:
Isn't this the purpose of "end_of_day" in the config? I have mine set to "03:00" so that it is considered 'today' until 3 am